tests/admin_scripts/tests.py

```diff
@@ -166,20 +185,26 @@ def safe_remove(path):
             except OSError:
                 pass
 
+        configured_settings_file = os.path.join(
+            os.path.dirname(__file__), 'configured_settings_manage.py')
+        conf_dir = os.path.dirname(conf.__file__)
+        if configured_settings and not os.path.isfile(configured_settings_file):
+            self.write_configured_settings()
+            self.addCleanup(self.remove_configured_settings)
         template_manage_py = (
-            os.path.join(os.path.dirname(__file__), 'configured_settings_manage.py')
-            if configured_settings else
-            os.path.join(os.path.dirname(conf.__file__), 'project_template', 'manage.py-tpl')
+            configured_settings_file if configured_settings else
+            os.path.join(conf_dir, 'project_template', 'manage.py-tpl')
         )
         test_manage_py = os.path.join(self.test_dir, 'manage.py')
         shutil.copyfile(template_manage_py, test_manage_py)
 
-        with open(test_manage_py, 'r') as fp:
-            manage_py_contents = fp.read()
-        manage_py_contents = manage_py_contents.replace(
-            "{{ project_name }}", "test_project")
-        with open(test_manage_py, 'w') as fp:
-            fp.write(manage_py_contents)
+        if not configured_settings:
+            with open(test_manage_py, 'r') as fp:
+                manage_py_contents = fp.read()
+            manage_py_contents = manage_py_contents.replace(
+                "{{ project_name }}", "test_project")
```

Comment:
Use single quotes (unless a string contains a single quote) as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).